### PR TITLE
New method to get the geocoder used quota from Redis

### PR DESCRIPTION
**Pull request by ethervoid**
- [x] New methods to retrieve the current used geocoder quota using the new geocoder metrics system based in Redis
- [x] Rake task to compare the geocoder quota values between the legacy (DB) and the new (Redis) geocoder metrics system 

// @rafatower please check this out

### CODE REVIEW

File: lib/tasks/geocoder_metrics.rake [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        quota_delta_percentage = Float(args[:quota_delta]) / 100
+      rescue
+        raise "Quota value passed is not an integer"
+      end
+      users = ::User.overquota(quota_delta_percentage)
+      puts "There is no overquota users for today" if users.blank?
+      users.each do |user|
+        legacy_quota_value = user.get_geocoding_calls
+        new_system_quota_value = user.get_new_system_geocoding_calls
+        if legacy_quota_value != new_system_quota_value
```
Comment (rafatower) [inline lib/tasks/geocoder_metrics.rake]: ok, let's run this as it is and we'll see later on if we need to add a tolerance for the checks :D

**Comment by rafatower** [comment]
Looks neat! :+1:

File: lib/tasks/geocoder_metrics.rake (hunk 1 above)
Comment (ethervoid) [inline lib/tasks/geocoder_metrics.rake]: yup i thought the same

**Comment by ethervoid** [comment]
retest this please

**Pull request merged**